### DISCUSSION TITLE
Drop-down calendar

### DISCUSSION THREAD

**Discussion by stephaniesaab** [CSC207-2022F-UofT/course-project-my-dog-ate-my-hw#28] [Ideas]
When users add a task and they have the option to pick a deadline, it may be easier to pick a day from a calendar than to type it out. We could add a drop-down menu with a calendar and then they may pick the day and type in the time, or they may pick a day and it will show multiple time slots.

**Comment by e-laflamme** [comment]
Instead of making a calendar graphic, I was thinking we could just have a drop down menu for each atrribute of time and date. So a drop down menu with the year (2022, 2023....), month, (Jan, Feb.....), and days in the month (1, 2, 3 ....). Then time could be typed in or also have time slots maybe? 

I will look into how to make an interactive calendar popup though. It would probably be easier for the user.
